packages/web-components/src/stories/astro-uxds/welcome/vue3.stories.mdx

```diff
@@ -42,23 +35,17 @@ This will be listed for each component in their documentation.
 
 ```js
```

Comment:
@markacianfrani I just inferred as best I could how the cherry picking should look, let me know what should be changed.